src/test/resources/features/automationPractice/ContactUs.feature

```diff
@@ -5,7 +5,14 @@ Feature: Contact Us
     When do click in the contact us button
     Then the CUSTOMER SERVICE from must be displayed
 
-  Scenario: new scenario for new myFirstBranch
-    Given 1111111
-    When 22222222
-    Then 3333333
\ No newline at end of file
+  Scenario:	Verify that a description message is displayed when the user select the subject type CUSTOMER SERVICE
+    Given The user is in the CONTACT US section
+    When The user click on subject heading control
+    And The user select the subject type "Customer service"
+    Then Verify that description message "For any question about a product, an order" must be displayed
+
+  Scenario: Verify that a description message is displayed when the user select the subject type WEBMASTER
+    Given The user is in the CONTACT US section
+    When The user click on subject heading control
+    And The user select the subject type "Webmaster"
+    Then Verify that description message "If a technical problem occurs on this website" must be displayed
```

Comment:
Add a new empty line at the end of the file

Reply:
A new empty line have been added in attention to your request.